src/Assert.php

```diff
@@ -1875,17 +1875,24 @@ public static function isList($array, $message = '')
             );
         }
 
-        if ($array === \array_values($array)) {
-            return;
-        }
-
-        $nextKey = -1;
-        foreach ($array as $k => $v) {
-            if ($k !== ++$nextKey) {
+        if (\function_exists('array_is_list')) {
+            if (!\array_is_list($array)) {
                 static::reportInvalidArgument(
                     $message ?: 'Expected list - non-associative array.'
                 );
             }
+            return;
+        }
+
+        if ([] === $array) {
+            return;
+        }
+
+        $keys = array_keys($array);
+        if (array_keys($keys) !== $keys) {
```

Comment:
Try running `composer run cs-fix`

Reply:
I get "The system cannot find the path specified."

Reply:
I've downloaded it and run it manually. Is this what you meant? It changed the array style to the old style.

Reply:
`composer run install-tools` first.

Reply:
Thanks, same outcome though. It still uses the old array syntax.